README.md

```diff
@@ -1,2 +1,6 @@
-# awesome-hicetnunc
-Awesome list of things in the broader hic et nunc ecosystem
+# awesome-hicetnunc [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
+<img align="right" width="100" height="100" src="https://raw.githubusercontent.com/yordanhristovUX/awesome-hicetnunc/bb2bac00cb8a36d2f9e541cd7ff56a5415e82eae/logoawehen.svg">
```

Comment:
now that #11 is merged please point this url to the relative local location of the logo

thank you!

Reply:
please dm me on twitter / discord :)

Reply:
Hi, was not on my PC, I think I put relative location, but not sure if it is right. Twitter don't let me message you and I don't have you discord. I am @brutalisti at Twitter, probably if you follow me, I will be able to message.